### PR TITLE
[HP-54] [HP-55] Closed POD Homepage And Child Pages

### PR DESCRIPTION
**Pull request by dchukhin**
https://caktus.atlassian.net/browse/HP-54
https://caktus.atlassian.net/browse/HP-55

This pull request:
 - adds a new `auth_content` app
 - adds a `ClosedPODHomePage` page
 - adds a `ClosedPODChildPage` page for `ClosedPODHomePage` children
 - adds a `get_closedpod_home_page_url()` utility function to return the ClosedPODHomePage URL
 - adds an `authenticated_view_router` to redirects a user when logging in (users in the "Closed POD" `Group` get redirected to the closed POD homepage, and all other users get redirected to the homepage)
 - updates the left sidebar to show different options when the user is on a Closed POD page

Note for testing: users are only redirected to the Closed POD home page if they are in the "Closed POD" `Group`. To test this functionality, add yourself to the "Closed POD" Django `Group` (through the wagtail interface or command line).

### CODE REVIEW

File: apps/hip/tests/test_views.py [hunk 1]
```diff
@@ -41,27 +42,120 @@ def test_hip_login_view_post_do_not_remember_user(db, client, remember_me):
...
+    response = client.get(reverse("auth_view_router"))
+
+    # GETting the auth_view_router redirects the user to the login URL.
+    assert 302 == response.status_code
+    assert f"{reverse('login')}?next={reverse('auth_view_router')}" == response.url
+    assert mock_get_home_page_url.called is False
+
+
+@pytest.mark.parametrize(
+    "group_names,expected_redirect",
```
Comment (vkurup) [inline apps/hip/tests/test_views.py:115]: nice!

**Review by vkurup — APPROVED** [review]
A few comments are included inline. I like the router!

I can successfully add myself to the closed POD group, create a closed POD homepage and then be able to see the closed POD homepage. But when I click the 'Back to public site' button, I don't see a left nav section to allow me to get back to the closed POD. Is that coming in a different ticket? (I'm fine with that work being done in a different PR)

:closed_book:

File: apps/auth_content/models.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        max_length=255,
+        blank=True,
+        help_text="The title of the 'About' section.",
+    )
+    about_text = RichTextField(
+        blank=True,
+        help_text="The text of the 'About' section.",
+    )
+
+    content_panels = [
```
[the remark this answers is not in the document]
Comment (dchukhin) [reply apps/auth_content/models.py:51]: Yes!

File: apps/auth_content/models.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+class ClosedPODChildPage(StaticPage):
+    parent_page_types = ["auth_content.ClosedPODHomePage"]
+
+    description = RichTextField(
+        blank=True,
+        help_text="The description that will appear for this page on the ClosedPODHomePage.",
+    )
+
+    content_panels = [
```
[the remark this answers is not in the document]
Comment (dchukhin) [reply apps/auth_content/models.py:101]: Yes!

**Comment by dchukhin** [comment]
Thanks for the review! For the sake of moving this work along, let's add the sidebar links in a different pull request

**Pull request merged**